ProcessingShowPath/Reader.pde

```diff
@@ -0,0 +1,65 @@
+class Reader{ 
+  private String[] filenames;
+  
+  Reader()
+  {
+    
+  }
+  
+  //get filenames from a specified folder and print them
+  void getFilesFromFolder(String fileFolderPath)
+  {
+    println("Files found: ");
+    filenames = listFileNames(fileFolderPath);
+    printArray(filenames);
+  }
+  
+  //read all game files from the folder the reader was created with
+  Path[] readGameFiles(String folder)
+  {
+    Path[] playerPaths = new Path[filenames.length];
+    for(int i = 0; i < filenames.length; i++)
+    {
+        playerPaths[i] = readTestFile(folder + "/" + filenames[i]);
+    }
+    return playerPaths;
+  }
+  
+  Path readTestFile(String filePath)
+  {
```

Comment:
both of the function names are inconsistent. Let's name them `readTestFile` and `readTestFileArray` maybe? 'Array', so the name wouldn't have the difference of only 1 letter)

Reply:
or maybe name it in a way for what it returns? Something like `getPlayerPath` and `getPlayerPathArray`?